WayMovies/View/DetailView.swift

```diff
@@ -0,0 +1,206 @@
+//
+//  DetailView.swift
+//  WayMovies
+//
+//  Created by Hersh Nagpal on 1/28/20.
+//  Copyright © 2020 Hersh Nagpal. All rights reserved.
+//
+
+
+import Foundation
+import UIKit
+
+class DetailView: UIView {
+    // The height of the main poster image
+    private let imageHeight:CGFloat = 630
+    
+    // The height of labels in the page
+    private let labelHeight:CGFloat = 40
+    
+    // The width of labels in the page
+    private let labelWidth:CGFloat = 170
+    
+    // The view showing the poster image
+    let imageView:UIImageView = {
+        let image = UIImageView()
+        image.translatesAutoresizingMaskIntoConstraints = false
+        image.contentMode = .scaleAspectFill
+        image.clipsToBounds = true
+        image.backgroundColor = .gray
+        return image
+    }()
+
+    // The label showing the description of the item in the detail page
+    let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .left
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        // Filler text
+        label.text = "ads i;fl vdksjf; lkad s kfl ;advs kjlfj l;sf; ;slda kjf;ls adkj f;lads kjf;alsd kf ;lka ds; lkf ja sd klfj ads;lkf jas ;lkfj ds; lkjfs; dkljf a;kljf ;als dkj f;l ksd jfl ;akds f;k las jfl;ka s djf;l akds jf;lk adsf"
+        label.numberOfLines = 0
+        label.sizeToFit()
+        return label
+    }()
+    
+    // The label showing the title of the movie or show or the name of the actor
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .left
+        label.textColor = .white
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    // The label showing whether the item is a show, movie, or actor
+    let typeLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.textColor = blue
+        label.font = label.font.withSize(40)
+        let insets = UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5)
+        label.drawText(in: CGRect().inset(by: insets))
```

Comment:
`https://developer.apple.com/documentation/uikit/uilabel/1620527-drawtext` Apple doc suggests that we should not call this method directly